### DISCUSSION TITLE
Brak wiedzy o kosztach w subskrybcji zarządzanej przez zewnętrznego partnera

### DISCUSSION THREAD

**Discussion by rmaziarka** [AzureCommunityPL/group#160] [Pomoc] [answered]
Cześć wszystkim,

Wykorzystuję subskrybcję Azure dostarczoną mi przez firmę partnerską w modelu CSP, ja jestem tam Ownerem. Wchodząc na daną subskrybcję nie widzę żadnych jej kosztów - żadnego podliczenia, nie mówiąc nawet o kosztach za poszczególne zasoby. Po spytaniu o to partnera dostałem odpowiedź, że nie mają takiej możliwości by pokazać zużycia kosztów.

Jestem przekonany, że będąc w poprzedniej firmie miałem analogiczną sytuację z partnerem, a mogłem podejrzeć pełen Billing dla subskrybcji. Ale może się mylę?

Obrazek poglądowy z obecnej subskrybcji, nie ma żadnych informacji + dodatkowe elementy menu (jak Cost Management) są ukryte:
![image](https://user-images.githubusercontent.com/5219374/100231602-8f56ea00-2f27-11eb-8233-e7b8f9185082.png)

**Comment by wi5nia** [comment]
Chyba Cie oszukuja ;) [tu](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/view-all-accounts) i [tu](https://docs.microsoft.com/en-us/azure/cost-management-billing/understand/mpa-overview) masz powyjasniane co i jak.

**Reply by smereczynski** [reply]
Niestety mylisz się :)

**Reply by wi5nia** [reply]
W sumie racja 👍🏻

**Comment by smereczynski** [comment, marked as answer]
To jest CSP w indirect. Nie zobaczysz tych danych. Nie da rady. Możesz zmienić na Azure Plan (będzie drożej dla partnera o jakieś 3pp). Wtedy zobaczysz koszty Pay as You Go bez uwzględnienia rabatu resellera.

**Reply by rmaziarka** [reply]
Dzięki Michał. Dowiedziałem się, że część partnerów ma własne platformy do sprawdzania kosztów więc chyba muszę takiego partnera wybrać 😉
